game/scripts/npc/abilities/invoker_cold_snap.txt

```diff
@@ -46,7 +46,7 @@
       "02"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "freeze_duration"                                 "0.4"
+        "freeze_duration"                                 "0.35" //OAA
```

Comment:
Not a fan of this change. It nerfs the spell at dota levels even if its only minor. Cold snap is only broken at OAA levels. 

Just make the freeze cooldown not scale into OAA levels and it wont perma stun. You only need a small interrupt to cast a lotus orb or bkb on yourself.

Reply:
This was an easier nerf suggested by Max.